package.json

```diff
@@ -1,16 +1,17 @@
 {
-  "name": "bugsahoy",
-  "version": "1.0.0",
-  "main": "index.js",
-  "license": "MIT",
+  "name": "bug-issue-ui",
+  "version": "0.0.0-private",
+  "private": true,
+  "license": "MPL-2.0",
   "scripts": {
     "build": "neutrino build",
     "start": "neutrino start",
-    "lint": "neutrino lint"
+    "lint": "neutrino lint",
+    "precommit": "neutrino stage"
   },
   "dependencies": {
-    "js-yaml-loader": "^0.1.0",
-    "material-ui": "^1.0.0-beta.41",
+    "@material-ui/core": "^1.1.0",
+    "@material-ui/icons": "^1.1.0",
```

Comment:
Does this break the netlify rendering? I remember @fiennyangeln mentioning that `@material-ui/core` and `@material-ui/icons` were removed because they were causing Netlify not to render properly. I do however agree we need to upgrade `material-ui` to use the stable version.

Reply:
Yeah, it seems to work fine.